src/main/index.ts

```diff
@@ -1,12 +1,16 @@
 import { app, dialog } from 'electron';
 import { autoUpdater } from 'electron-updater';
+import * as process from 'process';
 
 import { MainWindow } from './main';
 import { makeAppWithSingleInstanceLock } from './factories/instance';
 import { makeAppSetup } from './factories/setup';
 
-makeAppWithSingleInstanceLock(async () => {
-  app.commandLine.appendSwitch('autoplay-policy', 'no-user-gesture-required');
+const setupAutoUpdate = () => {
+  if (process.env.BUILD_SOURCE !== 'github') {
```

Comment:
We usually make fast return this way 😉

```suggestion
  if (process.env.BUILD_SOURCE !== 'github') return;
```